lang/cs/Org.Apache.REEF.Network/StreamingCodec/IDataCodecObj.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Org.Apache.REEF.Wake.Remote;
+
+namespace Org.Apache.REEF.Network.StreamingCodec
```

Comment:
Why not put those inside Group/Codec rather than create a new folder StreamingCodec? Are they mainly used for group communication? Anywhere else?